src/components/Reown/NanoContract/NanoContractActions.js

```diff
@@ -96,10 +115,22 @@ export const NanoContractActions = ({ ncActions, tokens, error }) => {
  * }} props.action A transaction's action object
  * @param {boolean} props.isNft A flag to inform if the token is an NFT or not
  * @param {string} props.title The card title for the action
+ * @param {Object} props.tokens Map of token metadata
```

Comment:
This is just a detail but we usually use token metadata as the info we download regarding NFTs

```suggestion
 * @param {Object} props.tokens Map of token details
```